datacenter/ucp/2.0/guides/configuration/integrate-with-ldap.md

```diff
@@ -8,6 +8,9 @@ title: Integrate with LDAP
 Docker UCP integrates with LDAP services, so that you can manage users from a
 single place.
 
+**Note:** When you switch from the built-in authentication to using LDAP
```

Comment:
I've addressed most of the feedback here. I've not changed from 'manually created' to 'local' because we use that term consistently in more places.